benchexec/tools/aise.py

```diff
@@ -29,7 +29,7 @@ def name(self):
         return "AISE"
 
     def project_url(self):
-        return "https://github.com/ZhenWang233/AISE"
+        return "https://github.com/zbchen/aise-verifier.git"
```

Comment:
First, this should probably be the URL without `.git`.

Second, I noticed that the new GitHub repository contains even less content than the old one, and not even the tool. Are you sure you want to designated this as the main URL for your tool?

Reply:
@ZhenWang233 do you agree with this?

Reply:
Hello, [PhilippWendler](https://github.com/PhilippWendler). I apologize for any confusion. Since AISE doesn’t have a tool's webpage, we would like to provide a simple introduction to our tool via GitHub. To facilitate future updates, we would like to modify the GitHub link.

Reply:
I have already added aise.zip on [GitHub](https://github.com/zbchen/aise-verifier), and I will update it with information for the SV-COMP 2025 competition in the future.

Reply:
> @ZhenWang233 do you agree with this?

Yes, It's ok, I agree with this change.

Reply:
Then please just remove the `.git` from the URL.

Reply:
OK，the `.git`  has been deleted.